### PR TITLE
Fix endpoint IDs in docs

### PR DESCRIPTION
**Pull request by martinkolenic**
This was basically a mass search and replace operation that required some endpoint generation. I found out that there was a quite wild mixture of IDs that were correct, those that were incorrect and mixtures of the two even within the same file. There are some endpoint IDs that I did not find oisTitles or endpointNames for, so I did not dare touch them, but otherwise I've gone through almost 600 occurrences of the endpointID variable name and fixed everything. You don't necessarily need to recheck the generated IDs; I only included reviewers to let you know this is done. I meant to create the PR late Friday afternoon but I needed to end early on a quick notice so I spent the evening cleaning up the commit and getting this ready.

### CODE REVIEW

File: archived/airnode/v0.2/config-json.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ in the Airnode repo.
   "triggers": {
     "rrp": [
       {
-        "endpointId": "0xeddc421714e1b46ef350e8ecf380bd0b38a40ce1a534e7ecdf4db7dbc9319353",
+        "endpointId": "0x13dea3311fe0d6b84f4daeab831befbc49e19e6494c41e9e065a09c3c68f43b6",
```
Comment (Siegrift) [inline archived/airnode/v0.2/config-json.md:57]: We should avoid doing changes in `archived` folder (those are archived materials used by Warren)

File: docs/airnode/pre-alpha/tutorials/airnode-starter.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ Your deployed Airnode will use these funds to make the transaction that will cre
...
 
-`config.json`defines an [endpoint](../protocols/request-response/endpoint.md) named`coinMarketData`, whose [endpoint ID](../protocols/request-response/endpoint.md#endpointid) is `0xf466b8feec41e9e50815e0c9dca4db1ff959637e564bb13fefa99e9f9f90453c`. Endpoints are not publicly accessible by default, so you will have to make a transaction for this. Run the following to set your endpoint's [authorizers](../protocols/request-response/authorizer.md) to `[0x0000000000000000000000000000000000000000]`, which makes it [publicly accessible](../guides/provider/setting-authorizers.md#allow-all):
+`config.json`defines an [endpoint](../protocols/request-response/endpoint.md) named`coinMarketData`, whose [endpoint ID](../protocols/request-response/endpoint.md#endpointid) is `0x6db9e3e3d073ad12b66d28dd85bcf49f58577270b1cc2d48a43c7025f5c27af6`. Endpoints are not publicly accessible by default, so you will have to make a transaction for this. Run the following to set your endpoint's [authorizers](../protocols/request-response/authorizer.md) to `[0x0000000000000000000000000000000000000000]`, which makes it [publicly accessible](../guides/provider/setting-authorizers.md#allow-all):
```
Comment (Siegrift) [inline docs/airnode/pre-alpha/tutorials/airnode-starter.md:159]: We should avoid doing changes in `pre-alpha` folder. The endpoint ID convention changed and pre-alpha uses the old format.

File: docs/airnode/v0.2/grp-providers/guides/build-an-airnode/http-gateway.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ use
 | parameter         | in     | CURL options                                             |
 | ----------------- | ------ | -------------------------------------------------------- |
 | x-api-key         | header | `-H 'x-api-key: 8d890a46-799d-48b3-a337-8531e23dfe8e'`   |
-| endpointId        | path   | `0xf466b8feec...99e9f9f90453c`                           |
+| endpointId        | path   | `0x6db9e3e3d...c7025f5c27af6`                            |
```
Comment (Siegrift) [inline docs/airnode/v0.2/grp-providers/guides/build-an-airnode/http-gateway.md:97]: Good catch.

File: docs/airnode/v0.4/reference/deployment-files/config-json.md [hunk 4]
```diff
@@ -343,14 +343,14 @@ defined in an OIS.
 {
   "rrp": [
     {
-      "endpointId": "0xe1da7948e4dd95c04b2aaa10f4de115e67d9e109ce618750a3d8111b855a5ee5",
+      "endpointId": "0xd7ddc8ee64d6e540682ec844a5dd9737663ec3afe5751102eb4f966744751838",
       "oisTitle": "myOisTitle",
       "endpointName": "myEndpointName"
     }
   ],
   "rrp": [
```
Comment (Siegrift) [inline docs/airnode/v0.4/reference/deployment-files/config-json.md:351]: Btw. this doesn't look right. I think it should be `http` section (currently there are two rrps). Can you update that as well?

**Review by Siegrift — CHANGES_REQUESTED** [review]

**Review by wkande — APPROVED** [review]
Nice catches, endpointIds seem to change somehow. This cleans them up.

File: docs/airnode/v0.4/reference/deployment-files/config-json.md (hunk 4 above)
Comment (wkande) [reply docs/airnode/v0.4/reference/deployment-files/config-json.md:351]: @kolenic-martin @Siegrift 
I think the second `rrp` is supposed to be `http`. However I cannot remember if `http` was there for `v0.4`. I will remove the 2nd occurrence now using the editor within this PR.

Comment (wkande) [reply docs/airnode/v0.4/reference/deployment-files/config-json.md:351]: If it is determined it needs to be added back in let me know and I will do so later. Given this is an older version of Airnode edits are only needed for anything broken.

File: docs/airnode/pre-alpha/tutorials/airnode-starter.md (hunk 2 above)
Comment (wkande) [reply docs/airnode/pre-alpha/tutorials/airnode-starter.md:159]: I think in general `pre-alpha` should no longer be changed at all. It reflects a product of historical value only. I would rather remove it all together. I do not think it matters much at this point to worry about this.

File: docs/airnode/v0.2/grp-providers/guides/build-an-airnode/http-gateway.md (hunk 3 above)
Comment (wkande) [reply docs/airnode/v0.2/grp-providers/guides/build-an-airnode/http-gateway.md:97]: These are alway a pain in the you know what as they reference config file. Thanks @kolenic-martin for the catch.

**Pull request merged**